### DISCUSSION TITLE
Logo license?

### DISCUSSION THREAD

**Discussion by m1212e** [sequelize/sequelize#15542] [Help & Questions] [answered]
Hi,
under what license is the  sequelize logo/trademark? I'd like to mention that I worked with sequelizein my resume, including the logo? Is that allowed?

**Comment by WikiRik** [comment, marked as answer]
Hi,

Thanks for reaching out to us!
This is not something that we've recently discussed with the team but the project has a [MIT License](https://github.com/sequelize/sequelize/blob/main/LICENSE) and I see no reason to deviate from this for the logo. So as long as you mention that the logo is from sequelize you are free to do so. We actually are very happy to see that people add sequelize to their experience in portfolios and resumes.
